### ISSUE TITLE
UI controls not working very well on ios - NET7.0

### ISSUE THREAD

**Issue by ikeoriaku123** [dotnet/maui#11072]
### Description

I recently changed my target framework from .net6 to .net7.0 and noticed that the UI controls do not work in some parts of areas in the layout for ios. It works well for android. Both ios and android devices work well with target framework .net6.

I have uploaded the solution and screenshots. When I click on any UI controls using .net6, everything works well. When changed to .net7.0, some parts on the screen work and some parts do not work. For example, when clicking on the button below the forgot password, it does not work. But when I place the button above the forgot password, it works.

I am having issues with visual studio for mac publishing to github therefore I have included relevant files and pictures to this link https://github.com/ikeoriaku123/fixmaui

### Steps to Reproduce

1. Change target framework to .net7.0
2. Rebuild the project
3. Select the device on which to run - selected my iphone device
4. Run project
5. Select UI controls

### Link to public reproduction project repository

https://github.com/ikeoriaku123/fixmaui

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

6.0.424

### Affected platforms

iOS

### Affected platform versions

iOS 16

### Did you find any workaround?

No

### Relevant log output

```shell
None
```

**Comment by ikeoriaku123** [comment]
@Eilon @jsuarezruiz Please any solution to this issue

**Comment by jfversluis** [comment]
Please try .NET 8 which should have many bugfixes. In addition, the repo that is linked doesn't exist or is private.

Also I think reporting issues separately will work better to track them individually and get them resolved instead of a generic issue like that that is hard to parse.

Thanks!
